java-client/src/main/java/org/opensearch/client/transport/rest_client/RestClientTransport.java

```diff
@@ -32,6 +32,7 @@
 
 package org.opensearch.client.transport.rest_client;
 
+import org.apache.hc.core5.http.HttpStatus;
```

Comment:
This is still not right import, the `RestClientTransport` uses Apache HttpClient 4.x, please use `org.apache.http.HttpStatus`

Reply:
I did not see org.apache.http.HttpStatus in your dependency. Shall I add it explicitly?

Reply:
My apologies @chenqi0805 , this is my bad, for `main` this is the right import, we would need to change it to the one I mentioned when backporting to 2.x